### PR TITLE
Mule 12547: Fix test for Beta

### PR DESCRIPTION
**Pull request by lucianoRM**

### CODE REVIEW

File: modules/deployment-model-impl/src/test/java/org/mule/runtime/module/deployment/impl/internal/artifact/MavenClassLoaderModelLoaderTestCase.java [hunk 1]
```diff
@@ -57,20 +55,39 @@ public void before() {
 
   @Test
   public void noMavenConfiguration() throws Exception {
+    Map<String, String> properties = getMuleFreeSystemProperties();
+    GlobalConfigLoader.reset(); //Change local repository path
```
Comment (gsfernandes) [inline modules/deployment-model-impl/src/test/java/org/mule/runtime/module/deployment/impl/internal/artifact/MavenClassLoaderModelLoaderTestCase.java]: I will move this to the block inside the withSysteProperties but set the MULE_RUNTIME_CONFIG_MAVEN_REPOSITORY_LOCATION system property in the Map of properties to be used so you warranty that the localRepositoryLocation is a temp location when the test runs

File: modules/deployment-model-impl/src/test/java/org/mule/runtime/module/deployment/impl/internal/artifact/MavenClassLoaderModelLoaderTestCase.java [hunk 2]
```diff
@@ -57,20 +55,39 @@ public void before() {
...
+    testWithSystemProperties(properties, () -> {
+      mavenClassLoaderModelLoader.load(artifactFile, emptyMap(), APP);
+    });
   }
 
   @Test
   public void changeMavenConfiguration() throws Exception {
+
+    Map<String, String> properties = getMuleFreeSystemProperties();
+    GlobalConfigLoader.reset(); //Change local repository path
```
Comment (gsfernandes) [inline modules/deployment-model-impl/src/test/java/org/mule/runtime/module/deployment/impl/internal/artifact/MavenClassLoaderModelLoaderTestCase.java]: Same as previous comment.

File: modules/deployment-model-impl/src/test/java/org/mule/runtime/module/deployment/impl/internal/artifact/MavenClassLoaderModelLoaderTestCase.java [hunk 3]
```diff
@@ -57,20 +55,39 @@ public void before() {
...
     expectedException.expectCause(instanceOf(DependencyCollectionException.class));
-    mavenClassLoaderModelLoader.load(artifactFile, emptyMap(), APP);
+    testWithSystemProperties(properties, () -> {
+      mavenClassLoaderModelLoader.load(artifactFile, emptyMap(), APP);
+    });
   }
 
   @Test
   public void changeMavenConfiguration() throws Exception {
+
```
Comment (gsfernandes) [inline modules/deployment-model-impl/src/test/java/org/mule/runtime/module/deployment/impl/internal/artifact/MavenClassLoaderModelLoaderTestCase.java]: Remove extra line

**Comment by lucianoRM** [comment]
jenkins validate

**Pull request merged**